deploy/tasks/main.yaml

```
@@ -25,13 +25,13 @@
   file:
     path: '{{ deploy_helper.new_release_path }}'
     state: directory
-    mode: 0755
+    mode: "a+rwx,go-w"
```

On utilise des single quotes partout ailleurs, et en fait il n'y a même pas besoin de quotes dans ce cas.
```suggestion
    mode: a+rwx,go-w
```